portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -603,6 +603,94 @@ definition {
 			resourcePanels = "Portal Services,Sample QA app");
 	}
 
+	@description = "This is a use case for LRQA-57134."
+	test RefreshTokenTime {
```

Comment:
Missing `@priority`. This value should be based off of the criticality of this use case. My opinion is that this test is a `4`.

Reply:
Prio 4 added.